wondrous-app/components/Common/Sidebar/Common/hooks.tsx

```diff
@@ -0,0 +1,28 @@
+import { PERMISSIONS } from 'utils/constants';
+import { parseUserPermissionContext } from 'utils/helpers';
+import { useBoards } from 'utils/hooks';
+
+export const useCanManage = () => {
+  const { board } = useBoards();
```

Comment:
instead of boards can we use Sidebar layout context, in latest staging there's GET_USER_PERMISSION_CONTEXT fetch at the sidebar level. It's done so we don't have to fetch this on every single board and I think we can use it here. Just in case for a new type of board where there's no permissions on the board context level ( might not be needed for current implementation tho ) more like a suggestion

Reply:
This has been updated to use `useCreateEntityContext`. However, we still require `useBoards` to parse the permission for the specific org/pod.